### PR TITLE
SPEC-970 : Add list of commands that support readConcern

### PR DESCRIPTION
**Pull request by rathisekaran**

### CODE REVIEW

File: source/causal-consistency/causal-consistency.rst [hunk 1]
```diff
@@ -512,4 +516,5 @@ Changelog
 - 2017-09-28: Update spec to reflect that replica sets use $clusterTime also now
 - 2017-10-04: Added advanceOperationTime
 - 2017-10-05: How to handle default read concern
-- 2017-10-06: advanceOperation MUST NOT validate operationTime
\ No newline at end of file
+- 2017-10-06: advanceOperation MUST NOT validate operationTime
```
Comment (behackett) [inline source/causal-consistency/causal-consistency.rst]: advanceOperationTime

File: source/read-write-concern/read-write-concern.rst [hunk 2]
```diff
@@ -78,6 +78,15 @@ For naming and deviation guidance, see the `CRUD specification <https://github.c
     level: Optional<ReadConcernLevel | String>
   }
 
+The read concern option is available for the following operations: 
+- ``find`` command
+- ``aggregate`` command and the ``db.collection.aggregate()`` method
```
Comment (behackett) [inline source/read-write-concern/read-write-concern.rst]: Get rid of the method part. That's shell implementation leaked into the command docs. Also, need to make clear aggregate with $out stage isn't supported.

File: source/causal-consistency/causal-consistency.rst [hunk 3]
```diff
@@ -305,6 +305,10 @@ the ``ClientSession`` as the value of the ``afterClusterTime`` field of the
         }
     }
 
+For the list commands that support causally consistent reads, see `ReadConcern`_ spec.
```
Comment (ajdavis) [inline source/causal-consistency/causal-consistency.rst]: "list of commands", "see the \`ReadConcern\`_"

File: source/causal-consistency/causal-consistency.rst [hunk 4]
```diff
@@ -512,4 +516,5 @@ Changelog
 - 2017-09-28: Update spec to reflect that replica sets use $clusterTime also now
 - 2017-10-04: Added advanceOperationTime
 - 2017-10-05: How to handle default read concern
-- 2017-10-06: advanceOperation MUST NOT validate operationTime
\ No newline at end of file
+- 2017-10-06: advanceOperationTime MUST NOT validate operationTime
+- 2017-11-17 : Added link to ReadConcern spec which list commands that support readConcern
```
Comment (ajdavis) [inline source/causal-consistency/causal-consistency.rst]: "2017-11-17: Added link to ReadConcern spec which lists commands that support readConcern"

**Pull request merged**